.travis.yml

```diff
@@ -0,0 +1,15 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
+language: julia
+os:
```

Comment:
Should we test against Windows? Do we know if this package failed on Windows in the past?